.gitignore

```diff
@@ -24,6 +24,7 @@ mozillavpn-daemon
 translations/*.qm
 translations/*.pri
 linux/netfilter/netfilter.[ah]
+linux/netfilter/vendor/
```

Comment:
There are issues with the content of this PR. This is unrelated, right?

Reply:
This line in the `.gitignore` file is intended to be there. When we download the Go module's dependencies for offline building using the `go mod vendor`, it will download them all into this directory. We can treat it as generated content, so it shouldn't be committed to the repository.